src/renderer/atlas/atlas.vcxproj

```diff
@@ -27,6 +27,28 @@
     <ClInclude Include="AtlasEngine.h" />
   </ItemGroup>
   <ItemGroup>
+    <FxCompile Include="custom_shader_ps.hlsl">

```

Comment:
wait is this #3928?

Reply:
hmm i don't believe so... but perhaps close?

Reply:
Yeah not yet. This just compiles the shader into a more compact binary format which is included as a header.